### ISSUE TITLE
Cannot "Close" or "Delete" Athlete

### ISSUE THREAD

**Issue by rommeswi** [GoldenCheetah/GoldenCheetah#4215]
Golden Cheetah opens every time with the same athlete (coincidentally the one who is the first in the alphabet) but I can no longer close or delete this athlete. There is no "close" button in the athlete tab and the "delete" button is greyed out. I also tried to just delete the folder from the AppData but then GoldenCheetah just asked me to "upgrade" the athlete and recreated the folder. I also tried to rename the athlete but changing the "nickname" in the settings does not work. My version is V3.6-BETA with Build id 4009.

Issue tracker is **only** for Bugs and Features, please don't open issues for questions or technical support. Before to open a new issue please read the contributing guidelines (link below).

If you have questions, please read the FAQs and User's/Developer's Guide:
* FAQs - https://github.com/GoldenCheetah/GoldenCheetah/wiki/FAQ
* User's Guide - https://github.com/GoldenCheetah/GoldenCheetah/wiki/UG_Main-Page_Users-Guide
* Developer's Guide - https://github.com/GoldenCheetah/GoldenCheetah/wiki/Developers-guide

If you need help or technical support please use the forums:
* Users - https://groups.google.com/forum/#!forum/golden-cheetah-users
* Developers - https://groups.google.com/forum/#!forum/golden-cheetah-developers

**Comment by amtriathlon** [comment]
Please read the FAQs and use the forum if you still have doubts, links above…

**Comment by rommeswi** [comment]
Thank you for the information! I managed to remove the athlete by opening Golden Cheetah using the "--" command line parameter.

However, I think it should still be a feature that 1) any athlete can be closed or opened after starting the program 2) any athlete can be deleted after starting the program. It is certainly not intuitive that one needs to pass a command line argument or deactivate a seemingly unrelated option (open last opened athlete on startup) to do this.
